testing/trino-tests/src/test/java/io/trino/tests/TestLocalQueries.java

```diff
@@ -91,12 +91,12 @@ public void testRejectStarQueryWithoutFromRelation()
     @Test
     public void testDecimal()
     {
-        assertQuery("SELECT DECIMAL '1.0'", "SELECT CAST('1.0' AS DECIMAL)");
-        assertQuery("SELECT DECIMAL '1.'", "SELECT CAST('1.0' AS DECIMAL)");
-        assertQuery("SELECT DECIMAL '0.1'", "SELECT CAST('0.1' AS DECIMAL)");
-        assertQuery("SELECT 1.0", "SELECT CAST('1.0' AS DECIMAL)");
-        assertQuery("SELECT 1.", "SELECT CAST('1.0' AS DECIMAL)");
-        assertQuery("SELECT 0.1", "SELECT CAST('0.1' AS DECIMAL)");
+        assertQuery("SELECT DECIMAL '1.0'", "SELECT CAST('1.0' AS DECIMAL(10, 1))");
+        assertQuery("SELECT DECIMAL '1.'", "SELECT CAST('1.0' AS DECIMAL(10, 1))");
```

Comment:
Can we use `assertQuery(String)` now?

Reply:
No. `SELECT DECIMAL x` is not supported syntax in H2.